database/init_database.sql

```diff
@@ -34,7 +33,7 @@ CREATE TABLE IF NOT EXISTS meetings (
 
 CREATE TABLE IF NOT EXISTS attendance (
     "meeting" TEXT REFERENCES meetings(id),
-    "email"   TEXT REFERENCES users(email)
+    "user"    TEXT REFERENCES users(email)
```

Comment:
Note that "email" actually still needs to stay here. I fixed this in #11 - "user" is a reserved keyword in postgreSQL and weird things happen if we use it.